[2025-02-08T19:00:44.895+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T19:00:44.910+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T19:00:44.920+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T19:00:44.921+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T19:00:44.967+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-12-02 00:00:00+00:00
[2025-02-08T19:00:44.975+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=823) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T19:00:44.977+0000] {standard_task_runner.py:72} INFO - Started process 839 to run task
[2025-02-08T19:00:44.976+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-12-02T00:00:00+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpgn00v1_7']
[2025-02-08T19:00:44.978+0000] {standard_task_runner.py:105} INFO - Job 87: Subtask generate_account_dim_data
[2025-02-08T19:00:45.025+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-02T00:00:00+00:00 [running]> on host 2c1cd59afcc0
[2025-02-08T19:00:45.106+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-02T00:00:00+00:00'
[2025-02-08T19:00:45.108+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T19:00:45.125+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T19:00:45.126+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T19:00:45.139+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T19:00:45.139+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-12-02T00:00:00+00:00, execution_date=20241202T000000, start_date=20250208T190044, end_date=20250208T190045
[2025-02-08T19:00:45.192+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T19:00:45.201+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-08T20:40:43.566+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T20:40:43.584+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T20:40:43.593+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-02T00:00:00+00:00 [queued]>
[2025-02-08T20:40:43.594+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-02-08T20:40:43.612+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): generate_account_dim_data> on 2024-12-02 00:00:00+00:00
[2025-02-08T20:40:43.628+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1582) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-08T20:40:43.629+0000] {standard_task_runner.py:72} INFO - Started process 1609 to run task
[2025-02-08T20:40:43.637+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'account_dim_gen', 'generate_account_dim_data', 'scheduled__2024-12-02T00:00:00+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/account_dim_gen.py', '--cfg-path', '/tmp/tmpeeoq3mdz']
[2025-02-08T20:40:43.639+0000] {standard_task_runner.py:105} INFO - Job 239: Subtask generate_account_dim_data
[2025-02-08T20:40:43.689+0000] {task_command.py:467} INFO - Running <TaskInstance: account_dim_gen.generate_account_dim_data scheduled__2024-12-02T00:00:00+00:00 [running]> on host 7e959c1a2a8b
[2025-02-08T20:40:43.796+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Saleh' AIRFLOW_CTX_DAG_ID='account_dim_gen' AIRFLOW_CTX_TASK_ID='generate_account_dim_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-02T00:00:00+00:00'
[2025-02-08T20:40:43.798+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T20:40:43.826+0000] {logging_mixin.py:190} INFO - CSV file ./account_dim_large_data.csv with 50 row has been generated successfully!
[2025-02-08T20:40:43.826+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T20:40:43.837+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T20:40:43.838+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=account_dim_gen, task_id=generate_account_dim_data, run_id=scheduled__2024-12-02T00:00:00+00:00, execution_date=20241202T000000, start_date=20250208T204043, end_date=20250208T204043
[2025-02-08T20:40:43.894+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T20:40:43.912+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-08T20:40:43.914+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
